src/styles/blocks/star.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+  width: 96px;
+  margin-left: 18px;
+}
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it is not recommended to use absolute units like pixels for layout-related properties. Consider using relative units like em or rem.